src/main/app/src/app/shared/batch-progress/batch-process.service.spec.ts

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Impressive tests! Just wondering if this is not somewhat overkill for this specific functionality? Maybe not. You know best I guess. :-)